src/foundry/foundry.js/userTargets.d.ts

```diff
@@ -1,36 +1,38 @@
-/**
- * A subclass of Set which manages the Token ids which the User has targeted
- * @internal
- */
-declare class UserTargets extends Set<Token> {
-  constructor(user: UserTargets['user']);
-
-  user: User;
+import { ConfiguredDocumentClass } from '../../types/helperTypes';
 
+declare global {
   /**
-   * Return the Token IDs which are user targets
+   * A subclass of Set which manages the Token ids which the User has targeted.
+   * @see User#targets
    */
-  get ids(): string[];
+  class UserTargets extends Set<Token> {
+    constructor(user: UserTargets['user']);
 
-  /**
-   * @override
-   * @remarks
-   * Returns void, but Set<T>.add returns boolean
-   */
-  add(token: Token): any;
+    user: InstanceType<ConfiguredDocumentClass<typeof User>>;
 
-  /** @override */
-  clear(): void;
+    /**
+     * Return the Token IDs which are user targets
+     */
+    get ids(): string[];
 
-  /**
-   * @override
-   * @remarks
-   * Returns void, but Set<T>.delete returns boolean
-   */
-  delete(token: Token): any;
+    /**
+     * @override
+     * @remarks Returns void, but Set<T>.add returns boolean
```

Comment:
We can solve this issue in a similar way as we solve it for `Collection` improperly extending `Map`.